actors/common/src/main/java/org/sunbird/bean/ClaimStatus.java

```diff
@@ -0,0 +1,17 @@
+package org.sunbird.bean;
+
+public enum ClaimStatus {
+
+
+    CLAIMED(0),
```

Comment:
I think better if we can use enum with int and Stirng , that will help in reporting